domains/meteor.sono.json

```diff
@@ -7,7 +7,7 @@
         },
     
         "record": {
-            "CNAME": "getsono.github.io"
+            "CNAME": "meteor-store.github.io"
```

Comment:
You should complete your website first, I only see a header with the text "website"